app/api/canvas/canvas.api.js

```diff
@@ -0,0 +1,162 @@
+"use strict";
```

Comment:
I use it only for development purposes. 
Let say you have 3 files open: canvas.api, canvas.model, canvas.service.
removing postfix will just make it harder to switch between open tabs, you have to go through tabs to find out is it canvas api or not